pandas/core/ops/__init__.py

```diff
@@ -140,6 +141,8 @@ def fill_binop(left, right, fill_value):
             right = right.copy()
             right[right_mask & mask] = fill_value
 
+    left = ensure_wrapped_if_datetimelike(left)
+    right = ensure_wrapped_if_datetimelike(right)
```

Comment:
can likely be removed here, but would need to change .values to ._values in Series._binop

Reply:
Thanks, that seems to have worked